lib/tasks/test_replication.rake

```diff
@@ -5,15 +5,26 @@ namespace :test do
   namespace :replication do
     desc "Setup environment for replication specs"
     task :setup => :initialize do
-      EvmTestSetupReplication.new.execute
+      EvmTestSetupReplication.new.execute_setup
+    end
+
+    desc "Teardown environment for replication specs"
+    task :teardown => :initialize do
+      EvmTestSetupReplication.new.execute_teardown
     end
   end
 
   desc "Run all replication specs"
-  RSpec::Core::RakeTask.new(:replication => :initialize) do |t|
+  RSpec::Core::RakeTask.new(:replication => [:initialize, :replication_util]) do |t|
```

Comment:
@carbonin Can you confirm that replication and replication_util runs in a separate process?  I don't think it does.  I think the key is it runs in a different test suite so the configured transactional fixtures from one doesn't pollute the other.